lib/widgets.php

```diff
@@ -126,3 +126,58 @@ function gutenberg_legacy_widget_settings( $settings ) {
 	return $settings;
 }
 add_filter( 'block_editor_settings', 'gutenberg_legacy_widget_settings' );
+
+/**
+ * Registers a wp_area post type.
+ */
+function gutenberg_create_wp_area_post_type() {
+	register_post_type(
+		'wp_area',
+		array(
+			'labels'       => array(
+				'name'                     => _x( 'Block Area', 'post type general name', 'gutenberg' ),
+				'singular_name'            => _x( 'Block Area', 'post type singular name', 'gutenberg' ),
+				'menu_name'                => _x( 'Block Areas', 'admin menu', 'gutenberg' ),
+				'name_admin_bar'           => _x( 'Block Area', 'add new on admin bar', 'gutenberg' ),
+				'add_new'                  => _x( 'Add New', 'Block', 'gutenberg' ),
+				'add_new_item'             => __( 'Add New Block Area', 'gutenberg' ),
+				'new_item'                 => __( 'New Block Area', 'gutenberg' ),
+				'edit_item'                => __( 'Edit Block Area', 'gutenberg' ),
+				'view_item'                => __( 'View Block Area', 'gutenberg' ),
+				'all_items'                => __( 'All Block Areas', 'gutenberg' ),
+				'search_items'             => __( 'Search Block Areas', 'gutenberg' ),
+				'not_found'                => __( 'No block area found.', 'gutenberg' ),
+				'not_found_in_trash'       => __( 'No block areas found in Trash.', 'gutenberg' ),
+				'filter_items_list'        => __( 'Filter block areas list', 'gutenberg' ),
+				'items_list_navigation'    => __( 'Block areas list navigation', 'gutenberg' ),
+				'items_list'               => __( 'Block areas list', 'gutenberg' ),
+				'item_published'           => __( 'Block area published.', 'gutenberg' ),
+				'item_published_privately' => __( 'Block area published privately.', 'gutenberg' ),
+				'item_reverted_to_draft'   => __( 'Block area reverted to draft.', 'gutenberg' ),
+				'item_scheduled'           => __( 'Block area scheduled.', 'gutenberg' ),
+				'item_updated'             => __( 'Block area updated.', 'gutenberg' ),
+			),
+			'public'       => false,
+			'show_ui'      => false,
+			'show_in_menu' => false,
+			'show_in_rest' => true,
+			'capabilities' => array(
+				// You need to be able to edit posts, in order to read blocks in their raw form.
+				'read'                   => 'edit_posts',
+				// You need to be able to publish posts, in order to create blocks.
```

Comment:
These comments seem to be copy-paste from the registration of `'wp_block'`, but aren't representative of what the actual values are (i.e. you need to be able to edit theme options, not publish posts).

https://github.com/WordPress/wordpress-develop/blob/96649a5ad1a80cd8aaea15b955a48e292c406135/src/wp-includes/post.php#L293-L294

Are they necessary at all to include? At least need to be updated.

Reply:
I agree I don't think comments are required in this situation and I removed them.